sql-injection-demo/scripts/sql_injections.pxl

```diff
@@ -0,0 +1,141 @@
+# Copyright 2018- The Pixie Authors.
```

Comment:
so each pxl script (pxl+vis.json pair) needs to be in its own subdirectory, maybe instead of scripts/ we should rename to script/ 

or we could do

pxl_scripts/sql_injection_demo